test/k8s-integration/main.go

```diff
@@ -420,10 +430,23 @@ func handle() error {
 
 	// Run the tests using the testDir kubernetes
 	if len(*storageClassFiles) != 0 {
-		storageClasses := strings.Split(*storageClassFiles, ",")
+		applicableStorageClassFiles := []string{}
+		for _, rawScFile := range strings.Split(*storageClassFiles, ",") {
+			scFile := strings.TrimSpace(rawScFile)
+			if len(scFile) == 0 {
+				continue
+			}
+			if scFile == regionalPDStorageClass && !*isRegionalCluster {
+				continue // Regional PDs cannot be used in a zonal cluster.
```

Comment:
Is there a way to log a warning?

Reply:
it's actually expected as we don't have a separate run-k8s-integration script for the regional job.

I'll add a warning but it will be a little noisy.